### DISCUSSION TITLE
ServiceStack.Redis writes to multiple redis issue

### DISCUSSION THREAD

**Discussion by wurangbo** [ServiceStack/Discuss#129] [Q&A]
Hi，
 I want to write the same data to three independent redis servers. using PooledRedisClientManager.
add key="DefaultRedisServer" value="siteweb1!@192.168.4.40:6379,siteweb1!@192.168.4.41:6379,siteweb1!@192.168.2.228:6379"      
                                       
But data can only be written to the first two redis：siteweb1!@192.168.4.40:6379,siteweb1!@192.168.4.41:6379
The third redis server siteweb1!@192.168.2.228:6379 write does not take effect .

The code can be found in the following repository
https://github.com/kinshines/ServiceStack.Redis.Utility

**Comment by Layoric** [comment]
Hi @wurangbo , The `PooledRedisClientManager` doesn't duplicate writes, it supports failover scenarios like [with use of Redis Sentinel](https://docs.servicestack.net/redis/sentinel). Only writing to one host is the documented behavior. If you want to write to multiple hosts, or handle multiple write hosts, currently that is something you would need to implement yourself.
